utility/src/main/res/values/dimens.xml

```diff
@@ -3,9 +3,10 @@
   <dimen name="bullet_radius">4dp</dimen>
   <dimen name="bullet_gap_width">16dp</dimen>
   <dimen name="bullet_y_offset">2dp</dimen>
-  <dimen name="bullet_x_offset">24dp</dimen>
-  <dimen name="bullet_leading_margin">20dp</dimen>
-  <dimen name="text_leading_margin">20dp</dimen>
+  <dimen name="spacing_before_bullet">20dp</dimen>
+  <!--  Space between parent (start point) and the bullet in a bullet span-->
```

Comment:
Comments should always appear *before* the thing they're describing. Please update.

Reply:
Done